src/pages/en/guides/imports.md

```diff
@@ -22,21 +22,23 @@ The following file types are supported out-of-the-box by Astro:
 - CSS Modules (`.module.css`)
 - Images & Assets (`.svg`, `.jpg`, `.png`, etc.)
 
-If you don't see the asset type that you're looking for, check out our [Integrations Library](https://astro.build/integrations/). You can extend Astro to add support for different file types, like Svelte and Vue components.
+Additionally, you can extend Astro to add support for different [UI Frameworks](/en/core-concepts/framework-components/) like React, Svelte and Vue components. You can also install the [Astro MDX integration](/en/guides/integrations-guide/mdx/) and use `.mdx` files in your project.
 
-This guide details how different types of assets are built by Astro, and how to import them successfully.
+### Files in `public/`
 
-Remember that you can also place any static asset in the [`public/` directory](/en/core-concepts/project-structure/#public) of your project, and Astro will copy them directly into your final build. `public/` files are not built or bundled by Astro, which means that any type of file is supported. You can reference a `public/` file by a URL path directly in your HTML templates.
+You can place any static asset in the [`public/` directory](/en/core-concepts/project-structure/#public) of your project, and Astro will copy it directly into your final build untouched. `public/` files are not built or bundled by Astro, which means that any type of file is supported. You can reference a `public/` file by a URL path directly in your HTML templates.
 
-## JavaScript
+## ESM import statements
+
+### JavaScript
 
 ```js
 import { getUser } from './user.js';
 ```
 
 JavaScript can be imported using normal ESM `import` & `export` syntax. This works as expected, based on default Node.js and browser behavior.
```

Comment:
```suggestion
JavaScript can be imported using normal ESM `import` & `export` syntax.
```

I'm not sure we gain anything by the second sentence. I also think "default Node.js" behavior can be confusing, as Node doesn't use ESM by default.